packages/api-v2/src/student/student.controller.ts

```diff
@@ -71,6 +73,31 @@ export class StudentController {
     return student;
   }
 
+  @UseGuards(JwtAuthGuard)
+  @Patch("me/onboard")
+  async onBoard(
+    @Req() req: AuthenticatedRequest,
+    @Body() onboardStudentDto: OnboardStudentDto
+  ): Promise<UpdateStudentResponse> {
+    const uuid = req.user.uuid;
+    const updateResult = await this.studentService.update(uuid, {
+      ...onboardStudentDto,
+      isOnboarded: true,
```

Comment:
hmm just thinking - does it count as `onboarded` if a student only goes through one step of onboarding? Do we want to say they are only onboarded if all fields have been filled

Reply:
oh hm unless we only send this request at the very end of the onboarding then nvm

Reply:
Yeeeee!! we only send the last request at the very end, which is why in the dto all fields are required.